File: kutuphane_flutter_app/lib/Model/book_model.dart
import 'package:json_annotation/json_annotation.dart';

part 'book_model.g.dart';

@JsonSerializable()
class BookModel{
  final int Id;
  final String Isim;
  final String Yazar;
  final String Tur;
  final int Sayfa;
  final String Durum;

  BookModel(this.Id, this.Isim, this.Yazar, this.Tur, this.Sayfa, this.Durum);

  factory BookModel.fromJson(Map<String, dynamic> json) => _$BookModelFromJson(json);
  Map<String, dynamic> toJson() => _$BookModelToJson(this);
}

File: KutuphaneWebAPI/Models/BookModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace KutuphaneWebAPI.Models
{
    public class BookModel
    {
        public string Isim { get; set; }

        public string Yazar { get; set; }

        public string Tur { get; set; }

        public int Sayfa { get; set; }

        public string Durum { get; set; }

    }
}